saleor/product/models.py

```diff
@@ -273,15 +244,11 @@ class Product(SeoModel, ModelWithMetadata, PublishableModel):
         default=settings.DEFAULT_CURRENCY,
     )
 
-    price_amount = models.DecimalField(
-        max_digits=settings.DEFAULT_MAX_DIGITS,
-        decimal_places=settings.DEFAULT_DECIMAL_PLACES,
-    )
-    price = MoneyField(amount_field="price_amount", currency_field="currency")
```

Comment:
I guess `currency` field could be also dropped. It was used for the compound `price` field, but not sure if it is used anywhere else - we would have to check this.

Reply:
`currency` field is also used in `minimal_variant_price` field.